devel/fortio/README.md

```diff
@@ -6,10 +6,13 @@ The name fortio comes from greek φορτίο which is load/burden.
 
 ## Command line arguments
 
+Fortio can be and http or grpc load generator, gathering statistics using the `load` command, or start simple http and grpc ping servers with the `server` command or issue grpc ping messages using the `grpcping` command.
+
 ```
 $ fortio
-Φορτίο 0.2.0 usage:
-	fortio [flags] target
+Φορτίο 0.2.2 usage:
```

Comment:
depends on who is "we", I haven't really heard an argument why in 2017 text output we can't have unicode characters  ? (see below for why it's on purpose in the http output, helps testing) - obviously the command name is just "fortio", having to type the greek letters would be bad